### DISCUSSION TITLE
Button per app to switch to next output with one click

### DISCUSSION THREAD

**Discussion by btmura** [File-New-Project/EarTrumpet#679] [Ideas]
Thank you for your excellent app!

I have headphones and speakers.  It would be convenient to be able to switch between these two with two clicks like this:

1. Click on EarTrumpet icon on task bar.
2. Find row with browser app playing sound through speakers.
3. Click on "Switch Output" icon.
4. Browser than plays sound through headphones rather than speakers.

Clicking on "Switch Output" icon again would go back to speakers, because there are only two devices on the system in a list like [headphones, speakers]. So this button would rotate through the list.

Currently, you need to:

1. Click on EarTrumpet icon on task bar.
2. Find row with browser app playing sound.
3. Right click on the row.
4. Click on arrows icon.
5. Find and select device from menu.

**Comment by riverar** [comment]
Thanks! We have this in our backlog for the next major release. Stay tuned!

**Comment by riverar** [comment]
Question: Would setting up a hotkey that switches [Your app] to [Device X] or [Device Y] work for you too?

**Comment by DavidWnek** [comment]
Using a hotkey would be awesome, I've been looking for a way to switch app outputs from a streamdeck.

+1

**Comment by beppe9000** [comment]
Related: https://github.com/sirWest/AudioSwitch can be setup to cycle available output devices by right-clicking on the notificationa area icon. May be useful as reference code or to embed in eartrumpet somehow.
